webui/react/src/components/Markdown.tsx

```diff
@@ -0,0 +1,53 @@
+import { Tabs } from 'antd';
+import { default as MarkdownViewer } from 'markdown-to-jsx';
+import React from 'react';
+
+import css from './Markdown.module.scss';
+import Spinner from './Spinner';
+
+const { TabPane } = Tabs;
+const MonacoEditor = React.lazy(() => import('components/MonacoEditor'));
+
+interface Props {
+  editing?: boolean;
+  height?: string;
+  markdown: string;
+  onChange?: (editedMarkdown: string) => void;
+}
+
+const Markdown: React.FC<Props> = ({ editing=false, height='80vh', markdown, onChange }: Props) => {
```

Comment:
**style suggestion:** To give `Markdown` a bit more flexibility, I think `height: '100%'` would be a bit better. `80vh` can potentially interfere with views where there are headers (or footers, tabs, other vertical space hogs) that are forced to being larger than `20vh` due to limited space.